### PR TITLE
Adding correct example to `setLight` docs and improving docs types

### PR DESCRIPTION
**Pull request by SnailBones**
Also adding globe option to extrusion query debug page

 - [X] briefly describe the changes in this PR
 - [ ] include before/after visuals or gifs if this PR includes visual changes
 - [X] apply changelog label ('bug', 'feature', 'docs', etc) or use the label 'skip changelog'

### CODE REVIEW

File: src/ui/map.js [hunk 1]
```diff
@@ -2466,13 +2466,16 @@ class Map extends Camera {
...
      * Sets the any combination of light values.
      *
-     * @param {Object} light Light properties to set. Must conform to the [Mapbox Style Specification](https://www.mapbox.com/mapbox-gl-style-spec/#light).
+     * @param {LightSpecification} light Light properties to set. Must conform to the [Light Style Specification](https://www.mapbox.com/mapbox-gl-style-spec/#light).
      * @param {Object} [options] Options object.
      * @param {boolean} [options.validate=true] Whether to check if the filter conforms to the Mapbox GL Style Specification. Disabling validation is a performance optimization that should only be used if you have previously validated the values you will be passing to this function.
      * @returns {Map} Returns itself to allow for method chaining.
      * @example
-     * const layerVisibility = map.getLayoutProperty('my-layer', 'visibility');
-     * @see [Example: Show and hide layers](https://docs.mapbox.com/mapbox-gl-js/example/toggle-layers/)
```
Comment (karimnaaji) [inline src/ui/map.js:2475]: Interesting, so this was a totally unrelated example?

File: debug/extrusion-query.html [hunk 2]
```diff
@@ -20,6 +20,7 @@
         <label>Projection:</label>
         <select id="projName">
             <option value="mercator" selected>Mercator</option>
+            <option value="globe">Globe</option>
```
Comment (karimnaaji) [inline debug/extrusion-query.html:23]: :+1:

**Review by karimnaaji — APPROVED** [review]
Thanks! LGTM

File: src/ui/map.js (hunk 1 above)
Comment (SnailBones) [reply src/ui/map.js:2475]: Yup, seems to be a copy-paste mistake

**Pull request merged**